### PR TITLE
First Pull

### PR DESCRIPTION
**Pull request by andrew1102**
I did a lot.

### CODE REVIEW

File: ClipAudio.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from pydub import AudioSegment
+from google.cloud import speech_v1p1beta1
+from google.cloud.speech_v1p1beta1 import enums
+from google.cloud.speech_v1p1beta1 import types
+
+def read_file(self, filename):
```
Comment (agude) [inline ClipAudio.py]: Functions should have doc strings. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

File: ClipAudio.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  self.response.write('Reading the full file contents:\n')
+  gcs_file = gcs.open('')
+  contents = gcs_file.read()
+  gcs_file.close()
+
+file_name = "Ross_Fadely.flac"
+client = speech_v1p1beta1.SpeechClient()
+audio = types.RecognitionAudio(uri="gs://twimlmp3/"+file_name)
+
+startMin = 9
```
Comment (agude) [inline ClipAudio.py]: You're mixing `snake_case` and `lowerCamelCase` in your naming.

Generally python uses `snake_case` for variables and `SCREAMING_SNAKE` for "constants".

File: ClipAudio.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+startSec = 50
+
+endMin = 9
+endSec = 55
+
+# Time to miliseconds
+startTime = startMin*60*1000+startSec*1000
```
Comment (agude) [inline ClipAudio.py]: You should probably write a function to do this:

```python 
def timestamp_to_millisec(minute, sec):
  min_milli = 60 * 1000 * minute
  sec_milli = 1000 * sec
  return min_milli + sec_milli
```

But with a doc string of course.

Also python recommends space around operators like `*`.

File: Reshuffle.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
+from keras.models import Model
+from keras.layers import Input
+from keras.layers import LSTM
+from keras.layers import Dense
+from keras.utils.vis_utils import plot_model
+
+def Load_Data():
```
Comment (agude) [inline Reshuffle.py]: Doc string.

File: Reshuffle.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    with open('Transcripts/API/Episodes/Ross_Fadely/sam_6.txt','r') as f1:
+        with open('Transcripts/True/Episodes/episode_68/sam_6.txt','r') as f2:
+            encode = f1.read()
+            decode = f2.read()
+            return encode,decode
+
+encode,decode = Load_Data()
```
Comment (agude) [inline Reshuffle.py]: Space around `,`

File: Reshuffle.py [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+print('encode: '+str(encode))
+print('decode: '+str(decode))
+# configure
```
Comment (agude) [inline Reshuffle.py]: This block is really hard to read, I'd add some line breaks between blocks.

File: Transcripts/CleanAPITranscript.py [hunk 7]
```diff
@@ -0,0 +1,70 @@
+from nltk.tokenize import sent_tokenize
+import os
+import re
+import math
+
+def braces(test_str):
```
Comment (agude) [inline Transcripts/CleanAPITranscript.py]: What is this function doing?

Comment (agude) [reply Transcripts/CleanAPITranscript.py]: (I mean I've figured it out, but it's not clear. Doc string and maybe some comments.)

File: Transcripts/CleanAPITranscript.py [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+            ret += i
+    return ret
+
+guest = 'Ross_Fadely'
+
+with open('API/'+guest+'.txt', 'rb') as f:
+    data = f.read()
+    paragraphs = data.split(str.encode("\n\n"))
+    paragraph_sentence_list = []
+    for paragraph in paragraphs:
```
Comment (agude) [inline Transcripts/CleanAPITranscript.py]: There should definitely be some functions in here.

What is clearer?

```python
for paragraph in paragraphs:
  paragraph = ...
  paragraph = ...
  paragraph = ...
```

or

```python
for paragraph in paragraphs:
  clean_paragraph = paragraph_cleaner(paragraph)
```

File: Transcripts/CleanTrueTranscript.py [hunk 9]
```diff
@@ -0,0 +1,84 @@
+from nltk.tokenize import sent_tokenize
+import os
+import re
+
+def braces(test_str):
```
Comment (agude) [inline Transcripts/CleanTrueTranscript.py]: Duplicate code.

**Comment by andrew1102** [comment]
Hi Alex, thanks for the feedback! That branch didn't have the newest code. Right now, I'm working on consolidating the code.

**Pull request merged**